PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -230,7 +230,12 @@ def miniAOD_customizeCommon(process):
                   'RecoEgamma.PhotonIdentification.Identification.cutBasedPhotonID_Spring15_50ns_V1_cff',
                   'RecoEgamma.PhotonIdentification.Identification.mvaPhotonID_Spring15_25ns_nonTrig_V2p1_cff',
                   'RecoEgamma.PhotonIdentification.Identification.mvaPhotonID_Spring15_50ns_nonTrig_V2p1_cff']
```

Comment:
@slava77 @ikrav I would say not only added, but updated (why would we keep the 15 IDs?)